### ISSUE TITLE
Misleading error message of xmutil pwrctl

### ISSUE THREAD

**Issue by talhaHavadar** [Xilinx/xmutil#5]
Hello,

When I try to use the pwrctl command of xmutil on my KV260 as below, it gives a misleading message.
```
ubuntu@kria:~$ xmutil pwrctl --on
Unkown device: xmutil pwrctl supports K24/K26 only

ubuntu@kria:~$ xmutil pwrctl --status
Unkown device: xmutil pwrctl supports K24/K26 only

ubuntu@kria:~$ xmutil pwrctl --off
Unkown device: xmutil pwrctl supports K24/K26 only
```

Instead, it should report that I need to use sudo because when I use it with sudo  it works as expected.
```
ubuntu@kria:~$ sudo xmutil pwrctl --status
PL sequencer is off

ubuntu@kria:~$ sudo xmutil pwrctl --on
ubuntu@kria:~$ sudo xmutil pwrctl --status
PL sequencer is on

ubuntu@kria:~$ sudo xmutil pwrctl --off
ubuntu@kria:~$ sudo xmutil pwrctl --status
PL sequencer is off
```

**Comment by chkohn** [comment]
@vshpatel please take a look when you are back. I sent you email on this as well.

**Comment by chkohn** [comment]
@vshpatel if this is resolved, can you please comment and close it out?

**Comment by vshpatel** [comment]
Fixed via ca51be7efca6ec22d90ba058fabd8a995cd67632

**Comment by vshpatel** [comment]
Fixed via ca51be7efca6ec22d90ba058fabd8a995cd67632
